server/main.lua

```diff
@@ -69,31 +69,41 @@ exports('UpdateVehicleEntity', updateEntity)
 
 ---@class SetEntityOwnerQuery
 ---@field citizenId string
+---@field license string
 ---@field vehiclePlate string
 
 --- Update Vehicle Entity Owner
 ---@param query SetEntityOwnerQuery
 local function setEntityOwner(query)
-    MySQL.update('UPDATE player_vehicles SET citizenid = ? WHERE plate = ?', {
+    MySQL.update('UPDATE player_vehicles SET citizenid = ?, license = ? WHERE plate = ?', {
         query.citizenId,
+        query.license,
         query.vehiclePlate
     })
 end
 
 exports("SetVehicleEntityOwner", setEntityOwner)
 
---- Deletes a DB Vehicle Entity through searching for the number plate
----@param vehiclePlate string
-local function deleteEntityByPlate(vehiclePlate)
-    MySQL.query('DELETE FROM player_vehicles WHERE plate = ?', {vehiclePlate})
-end
-
-exports('DeleteVehicleEntityFromPlate', deleteEntityByPlate)
-
 --- Deletes DB Vehicle entities(-y) through searching for the citizen id
 ---@param citizenId string 
 local function deleteEntitiesByCitizenId(citizenId)
     MySQL.query('DELETE FROM player_vehicles WHERE citizenid = ?', {citizenId})
 end
 
 exports('DeleteVehicleEntitiesByCitizenId', deleteEntitiesByCitizenId)
+
+--- Deletes a DB Vehicle Entity through searching for the number plate
+---@param plate string
+local function deleteEntityByPlate(plate)
+    MySQL.query('DELETE FROM player_vehicles WHERE plate = ?', {plate})
+end
+
+exports('DeleteVehicleEntityByPlate', deleteEntityByPlate)
+
+--- Deletes DB Vehicle entities(-y) through searching for the license
+---@param license string 
+local function deleteEntitiesByLicense(license)
+    MySQL.query('DELETE FROM player_vehicles WHERE license = ?', {license})
+end
```

Comment:
Deleting a user's vehicles by license (all their vehicles across characters) doesn't seem like a usual use case. Should this be delete by citizenid instead?

Reply:
There already is a deleteEntitiesByCitizenId

Reply:
What's the use case for deleting all vehicles of a player across characters?

Reply:
Thats a great question...

Reply:
If we can't come up with a use case it's better not to have the export.